pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -134,6 +139,14 @@ func newEventConsumer(
 	return c, ss, nil
 }
 
+func defaultNumWorkers() int64 {
+	idealNumber := runtime.NumCPU() >> 2
+	if idealNumber < 1 {
+		return 1
+	}
+	return int64(idealNumber)
```

Comment:
let's cap this please; 
m6a.32xlarge has 132 vCPUs, I don't think we should be using 33 workers on this machine.  I do think a cap of 8 makes sense;
If you want 33 workers, set the setting to 33.